MSAL/test/app/ios/MSALTestAppSettingsViewController.m

```diff
@@ -133,24 +118,36 @@ - (void)viewWillAppear:(BOOL)animated
 {
     (void)animated;
     
-    // TODO: WPJ state
-    /*
-    ADRegistrationInformation* regInfo =
-    [ADWorkPlaceJoinUtil getRegistrationInformation:nil error:nil];
-    
-    NSString* wpjLabel = @"No WPJ Registration Found";
+    MSALTestAppSettingsRow* clientIdRow = [MSALTestAppSettingsRow rowWithTitle:MSAL_APP_CLIENT_ID];
+    NSDictionary *currentProfile = [MSALTestAppSettings currentProfile];
+    NSString *clientId = [currentProfile objectForKey:MSAL_APP_CLIENT_ID];
+    clientIdRow.valueBlock = ^NSString *{ return clientId; };
+    MSALTestAppSettingsRow* redirectUriRow = [MSALTestAppSettingsRow rowWithTitle:MSAL_APP_REDIRECT_URI];
+    NSString *redirectUri = [currentProfile objectForKey:MSAL_APP_REDIRECT_URI];
+    redirectUriRow.valueBlock = ^NSString *{ return redirectUri; };
     
-    if (regInfo.userPrincipalName)
-    {
-        wpjLabel = regInfo.userPrincipalName;
-    }
-    else if (regInfo)
+    MSIDRegistrationInformation* regInfo =
+    [MSIDWorkPlaceJoinUtil getRegistrationInformation:nil workplacejoinChallenge:nil];
+    _wpjState = @"No WPJ Registration Found";
+
+    if (regInfo)
     {
-        wpjLabel = @"WPJ Registration Found";
+        _wpjState = @"WPJ Registration Found";
```

Comment:
we have an API now to read deviceId in MSIDWorkPlaceJoinUtil, can we use that one here?

Reply:
Updated code to use the new api and display device info in settings view.